NEWS.md

```diff
@@ -363,6 +363,15 @@ Deprecated or removed
 
   * `diagm(A::BitMatrix)` has been deprecated, use `diagm(vec(A))` instead ([#23373]).
 
+  * `ℯ` (written as `\mscre<TAB>` or `\euler<TAB>`) is the new default for Euler's
```

Comment:
What does it mean for this to be "the new default"? Default in what sense?

Reply:
The one being exported, and also `typeof(e)` has changed from `Irrational{:e}` to `Irrational{:ℯ}`.

Reply:
Ok, might be good to clarify that if you're willing!

Reply:
Yes, I think my own wording is kinda weird too.